spec/schemas/Order.yml

```diff
@@ -0,0 +1,8 @@
+type: string
+description: |
+  Order (ascending or descending) based on the collection property set on the param ``orderBy``.
```

Comment:
i think this is outdated ^^

Reply:
Looks correct, I have added more details to the comment:

`If the request does not specify ``orderBy``, REST returns the collection ordered by id.
`